pkg/client/client.go

```diff
@@ -236,6 +267,11 @@ func (c *ConmonClient) startServer(config *ConmonServerConfig) error {
 }
 
 func (c *ConmonClient) toArgs(config *ConmonServerConfig) (entrypoint string, args []string, err error) {
+	_, span := c.startSpan(context.TODO(), "toArgs")
```

Comment:
tbh, I don't see how useful inspecting this span would be. I would vote it should be dropped

Reply:
Good, I removed that span.